cmd/utils/flags.go

```diff
@@ -8,6 +8,33 @@ import (
 )
 
 var (
+	// Titond flags
+	TitondFlags = []cli.Flag{
```

Comment:
we will get the flags from user later. I think we have to change this except `titond.namespace`, `titond.contracts.target.network` later. 🙇🏼‍♂️